### DISCUSSION TITLE
chord duration (length of the chord once played)

### DISCUSSION THREAD

**Discussion by sdlawsonsjlondonorg** [musescore/MuseScore#16414] [General]
issue chords bleeding into the next.  Example "intro of "Eye Of The Tiger" the bumps.  how can one fine tune the duration of each chord or note.  There are no sharp cut off at  the end of the chord it rings slightly into the next chord almost as if the 2 last chords are ringing together. 

Another example "I feel good" bump! bump! 
Another is I am working on a intro that walks up from 2-3-4=5
I need the 2, 3, 4 to barely sound Kind of of like bumps

**Comment by MarcSabatella** [comment]
In order for someone to understand what you mean and assist, please attach a score demonstrating the problem.  You can ZIP it first so GitHub will accept it.

In general, though, when asking for help using MuseScore, the official Support forum on musescore.org is the place.  GitHub is meant more for development discussions, not user support.

**Comment by sdlawsonsjlondonorg** [comment]
Thank you.     On Friday, February 17, 2023, 11:00:13 AM EST, Marc Sabatella ***@***.***> wrote:  
 
 


In order for someone to understand what you mean and assist, please attach a score demonstrating the problem. You can ZIP it first so GitHub will accept it.

In general, though, when asking for help using MuseScore, the official Support forum on musescore.org is the place. GitHub is meant more for development discussions, not user support.
